portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,39 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
+/*
+ * Adds a random quote to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomPrequelQuote() {
+  const quotes = [`Hello there.`, `There’s always a bigger fish.`, `I \
```

Comment:
The style guide forbids line continuations with a trailing '\': https://google.github.io/styleguide/jsguide.html#features-strings-no-line-continuations

In this case, it might make more sense to put each quote on its own line, and if a quote is too long for a line, use + to concatenate the strings.

Reply:
Fixed